### PR TITLE
Mutect3 dataset enhancements: optional truth VCF for labels, seq error likelihood annotation

### PR DESCRIPTION
**Pull request by davidbenjamin**

### CODE REVIEW

**Comment by davidbenjamin** [comment]
@meganshand Would you mind reviewing this PR, which will enable M3 testing on Ultima?  To the extent your conscience permits it really only needs a rubber stamp.  No one besides me is familiar with the M3 code and while that is eventually worth addressing I don't think you or anyone else should be responsible for increasing the M3 bus factor until it is more stable.

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java [hunk 1]
```diff
@@ -203,6 +204,14 @@ public double getDefaultAlleleFrequency() {
     @Argument(fullName = MUTECT3_DATASET_LONG_NAME, optional = true, doc="Destination for Mutect3 data collection")
     public File mutect3Dataset;
 
+    /**
+     * VCF of known calls for a sample used for generating a Mutect3 training dataset.  Unfiltered variants contained
```
Comment (meganshand) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java:208]: Unfiltered here meaning PASS or `.`?
```suggestion
     * VCF of known calls for a sample used for generating a Mutect3 training dataset.  Passing variants contained
```

**Review by meganshand — APPROVED** [review]
Thanks @davidbenjamin! Looks good 👍

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java (hunk 1 above)
Comment (davidbenjamin) [reply src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java]: done

Comment (davidbenjamin) [reply src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java]: It means either.

**Comment by davidbenjamin** [comment]
Thanks @meganshand!

**Pull request merged**